src/Workspaces/Core/Portable/Shared/Extensions/INamedTypeSymbolExtensions.cs

```diff
@@ -374,7 +374,7 @@ private static bool IsAbstractPropertyImplemented(INamedTypeSymbol classOrStruct
             ISymbol within)
         {
             var systemAttributeType = compilation.AttributeType();
-
+            var propertyNames = new HashSet<string>(StringComparer.Ordinal);
```

Comment:
Thanks I'll fix and make the test get this case

On Mon, 8 Aug 2016 13:08 0xd4d, <notifications@github.com> wrote:

> In
> src/Workspaces/Core/Portable/Shared/Extensions/INamedTypeSymbolExtensions.cs
> <https://github.com/dotnet/roslyn/pull/12982#discussion_r73863930>:
>
> > @@ -374,7 +374,7 @@ private static bool IsAbstractPropertyImplemented(INamedTypeSymbol classOrStruct
> >              ISymbol within)
> >          {
> >              var systemAttributeType = compilation.AttributeType();
> > -
> > +            var propertyNames = new HashSet<string>(StringComparer.Ordinal);
>
> compilation.IsCaseSensitive is probably what you need.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dotnet/roslyn/pull/12982/files/3479950d48893a913fcb4ba84575f06762e0e1ed#r73863930>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA822u55q65jPp7ZeRizRjQidsS7Rh-mks5qdxwLgaJpZM4JegCo>
> .
>